bitnami/etcd/Chart.yaml

```
@@ -24,4 +24,4 @@ maintainers:
 name: etcd
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/etcd
-version: 8.11.4
+version: 8.11.5
```

I think this should be a minor.
```suggestion
version: 8.12.0
```